src/test/java/org/fundacionjala/coding/dennis/CountChar.java

```diff
@@ -0,0 +1,7 @@
+package org.fundacionjala.coding.dennis;
+
+class CountChar {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)